src/searchengine/nova3/nova2.py

```diff
@@ -1,4 +1,4 @@
-#VERSION: 1.48
+# VERSION: 1.49
```

Comment:
IIRC, it is used for update purposes. Are you sure such change won't break it?

Reply:
> IIRC, it is used for update purposes. Are you sure such change won't break it?

I mean additional space after `#`.

Reply:
>IIRC, it is used for update purposes. Are you sure such change won't break it?

It is incompatible with the old parser but we already ship the new parser since qbt v4.1.x:
https://github.com/qbittorrent/qBittorrent/blob/v4_1_x/src/base/search/searchpluginmanager.cpp#L558-L559

And besides, qbt v4.6 has raised python requirement so older qbt won't able to support it anyway.
https://github.com/qbittorrent/qBittorrent/blob/v4_6_x/INSTALL#L21

All in all it shouldn't be a problem.